client/src/components/SignUp/SignUp.js

```diff
@@ -40,7 +40,11 @@ class SignUp extends Component {
       "email": this.state.email,
       "username": this.state.username,
       "password": this.state.password,
-      "passwordConfirm": this.state.passwordConfirm
+      "passwordConfirm": this.state.passwordConfirm,
+      // "stats":{
```

Comment:
I made these fields have default values of 0 in the database User model, so we don't need to send this data from the client. When a new user is created server side, it will automatically get these values.